pkg/domain/utils/utils.go

```diff
@@ -24,18 +45,568 @@ var DeepCopy = func(dst interface{}, src interface{}) error {
 	return nil
 }
 
-func ToLibpodFilters(f url.Values) (filters []string) {
+func ToLibpodFilters(f urlP.Values) (filters []string) {
 	for k, v := range f {
 		filters = append(filters, k+"="+v[0])
 	}
 	return
 }
 
-func ToURLValues(f []string) (filters url.Values) {
-	filters = make(url.Values)
+func ToURLValues(f []string) (filters urlP.Values) {
+	filters = make(urlP.Values)
 	for _, v := range f {
 		t := strings.SplitN(v, "=", 2)
 		filters.Add(t[0], t[1])
 	}
 	return
 }
+
+func ExecuteTransfer(args []string, parentFlags []string, quiet bool) (*entities.ImageLoadReport, *entities.ImageScpOptions, *entities.ImageScpOptions, []string, error) {
```

Comment:
Can you throw all the SCP functions in a separate file, same package (`pkg/domain/utils/scp.go` maybe?)